src/providers/RelayProvider.ts

```diff
@@ -102,4 +119,18 @@ export class RelayProvider implements TLProvider {
     }
     return this.fetchEndpoint<string>(`relay`, options)
   }
+
+  /**
+   * Send the given signed meta-transaction to a relay server to execute it on the
+   * blockchain and returns a `Promise` with the transaction hash.
+   * @param metaTransaction Signed meta-transaction to be sent to the relay server
+   * @returns The hash of the transaction sent by the relay server, not to be confused with the hash of the meta-transaction
+   */
+  public async sendSignedMetaTransaction(
+    metaTransaction: MetaTransaction
+  ): Promise<string> {
+    return this.postToEndpoint<string>('relay-meta-transaction', {
+      metaTransaction
```

Comment:
`signedMetaTransaction` ?

Reply:
I can use `signedMetaTransaction` but the relay server API expects `"metaTransaction"`.
So it will result in `return this.postToEndpoint<string>('relay-meta-transaction', { metaTransaction: signedMetaTransaction })`.
Do you think it is better?

Reply:
how other parts are named should not limit us. Yes I think this is better.